apispec-model/src/main/scala/sttp/apispec/Schema.scala

```diff
@@ -26,76 +26,147 @@ object AnySchema {
 
 // todo: xml
 case class Schema(
-    $ref: Option[String] = None,
+    // Core JSON Schema keywords
+    // https://json-schema.org/draft/2020-12/json-schema-core#section-8
     $schema: Option[String] = None,
+    $vocabulary: Option[ListMap[String, Boolean]] = None,
+    $id: Option[String] = None,
+    $anchor: Option[String] = None,
+    $dynamicAnchor: Option[String] = None,
+    $ref: Option[String] = None,
+    $dynamicRef: Option[String] = None,
+    $comment: Option[String] = None,
+    $defs: Option[ListMap[String, SchemaLike]] = None,
+
+    // General assertions
+    // https://json-schema.org/draft/2020-12/json-schema-validation#section-6.1
+    `type`: Option[List[SchemaType]] = None,
```

Comment:
the type can be a list? 😱

Reply:
I mean, there might be multiple types?

Reply:
Yeah, in particular that's how you express nullability in OAS 3.1

Reply:
Note: this was already expressible before my changes with `ArraySchemaType` (which I previously misunderstood for a schema type for an array).

Reply:
Ah yes, that's possible. But if we can have a simpler representation, that's better